compact/far_backward/lumi/exitWindow.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd>
+
+  <!-- Luminosity exit window -->
+
+  <define>
+    <constant name="LumiWin_gap" value="1*mm"/>
+  </define>
+
+  <detectors>
+    <detector
+      id="LumiWindow_ID"
+      name="LumiWindow"
+      type="LumiWindow"
+      material="Aluminum"
+      vis="Shape1_vis">
+      <dimensions x="LumiWin_DXY/2" y="LumiWin_DXY/2" z="LumiWin_thickness/2" />
+      <position   x="0" y="0" z="LumiWin_Z - LumiWin_thickness/2 - LumiWin_gap" />
```

Comment:
Why is there a LumiWin_gap?

Reply:
To prevent a geometry overlap error.

Reply:
Just tested this with a value of 0 and seemed to pass the geometry check, it shouldn't be needed. Maybe for an earlier version of the vacuum I'd extended the vacuum wall beyond LumiWin_Z.